### PR TITLE
(Fix) visualize if validator already voted

### LINKED ISSUE #63: (UI) indicate if you already voted for the ballot visually

### PR DESCRIPTION
**Pull request by vbaranov**
Closes https://github.com/poanetwork/poa-dapps-voting/issues/63

screencapture-localhost-3005-poa-dapps-voting-2018-03-22-15_26_52

Also,
- white color for an empty vote scale for an active ballot for better contrast
- fix for `to finalize` filter

### CODE REVIEW

File: src/components/BallotCard.jsx [hunk 1]
```diff
@@ -399,6 +414,9 @@ export class BallotCard extends React.Component {
         let { contractsStore, votingType, children, isSearchPattern } = this.props;
         let isFromSearch = (this.isCreatorPattern() || this.isMemoPattern() || isSearchPattern);
         let ballotClass = (this.showCard() && isFromSearch) ? this.isFinalized ? "ballots-i" : "ballots-i ballots-i-not-finalized" : "ballots-i display-none";
+        let voteScaleClass = this.isFinalized ? "vote-scale" : "vote-scale vote-scale-not-finalized";
+        let hasAlreadyVotedLabel = <div type="button" className="ballots-i--vote ballots-i--vote-label ballots-i--vote-label-right ballots-i--vote_no">You already voted</div>;
```
Comment (phahulin) [inline src/components/BallotCard.jsx]: Should this `div` be without `type="button"`?

File: src/components/BallotCard.jsx [hunk 2]
```diff
@@ -191,6 +192,18 @@ export class BallotCard extends React.Component {
         this.creator = fullName ? fullName : _miningKey;
     }
 
+    @action("validator has already voted")
+    getHasAlreadyVoted = async () => {
+        const { contractsStore, id, votingType } = this.props;
+        let _hasAlreadyVoted;
```
Comment (phahulin) [inline src/components/BallotCard.jsx]: As a minor note, `_hasAlreadyVoted` should be set to something by default (probably to `false` the same as in constructor)?

Comment (vbaranov) [reply src/components/BallotCard.jsx]: you're right, thanks

File: src/components/BallotCard.jsx (hunk 1 above)
Comment (vbaranov) [reply src/components/BallotCard.jsx]: yes, I'll remove

**Review by phahulin — APPROVED** [review]

**Pull request merged**